src/DependencyInjection/PsyshExtension.php

```diff
@@ -35,6 +35,16 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../../resources/config'));
         $loader->load('services.xml');
+        $paths = [
```

Comment:
this looks a bit weird, why does those two need to load another bundle resources?

Reply:
I agree, thanks for the response. 

The problem is following: service `Symfony\Bundle\FrameworkBundle\Test\TestContainer` by default available only in `test` environment and we need to enable it in `dev` environment. As discussed in the issue #36, there are 2 ways to register this service - load [test.xml](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/test.xml) in PsyshExtension or copy & paste service definitions to PsyshBundle. We chose the first option. But here is another problem, after installing Symfony using `composer create-project symfony/skeleton blog` file `test.xml` is located at path `vendor/symfony/framework-bundle/Resources/config/test.xml`. But after `git clone https://github.com/theofidry/PsyshBundle && composer install` file `test.xml` is located at path `/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Resources/config/test.xml`

One of the possible solution is to copy & paste `test.xml` to the PsyshBundle, what do you think?

Reply:
I see; the issue is that it relies on a Compiler pass as well: https://github.com/symfony/symfony/blob/2b9c142f07b7dbc11d4d392b77629fa91a5dcb41/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/TestServiceContainerRealRefPass.php

But I think copy/pasting the relevant content is a better idea. I would however add an additional check to see if the service is not already registered so that when Symfony registers it (e.g. in the `test` environment), we are using this one instead of registering ours/overriding it with our own.